profiles/ericspring08.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/69996843/176015434-b1911f22-fad9-4d2a-912e-08fd51e218f5.png",
+  "issueId": 1029,
```

Comment:
The issue number is wrong. I believe it should be 1209.
```suggestion
  "issueId": 1209,
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable